protractor-typescript-cucumber/Config/config.ts

```diff
@@ -0,0 +1,41 @@
+import {ProtractorBrowser} from 'protractor';
+/*
+The config folder includes all the configuration files
+This example config file displays the basic protractor-cucumber framework configuration
+ts-node compiler is needed for cucumberjs
+tags option for specific scenarios added
+**/
+export let config = {
+
+    directConnect: true,
+
+    baseUrl: 'http://juliemr.github.io/protractor-demo/',
```

Comment:
@igniteram applied changes with commit https://github.com/angular/protractor-cookbook/pull/14/commits/af051af2605541e55d0d714bd18c9add7fcd21f9. Added a `environment.ts` file which should take care of the environment changes and this could be scaled to further test apps here like Angular 2 test app in future! 👍